datauri/__init__.py

```diff
@@ -169,6 +169,16 @@ def validate(cls, v: str) -> Self:
             raise ValueError("invalid data-uri format")
         return m
 
+    @classmethod
+    def __get_pydantic_json_schema__(cls, core_schema: Any, handler: Any) -> Any:
```

Comment:
shouldn't `core_schema` have type `Mapping[str, Any]`, as in [pydantic_core](https://github.com/pydantic/pydantic-core/blob/main/python/pydantic_core/core_schema.py#L3855C29-L3855C46)?

Reply:
Yes, added mapping Dict